consumer/processing.go

```diff
@@ -161,6 +161,28 @@ func writeRecommendations(
 	return tStored, nil
 }
 
+func (consumer *KafkaConsumer) writeInfoReport(
+	msg *sarama.ConsumerMessage, message incomingMessage, infoStoredAtTime time.Time,
+) error {
+	err := consumer.Storage.WriteReportInfoForCluster(
+		*message.Organization,
+		*message.ClusterName,
+		message.ParsedInfo,
+		infoStoredAtTime,
+	)
+	if err != nil {
+		if err == types.ErrOldReport {
+			logMessageInfo(consumer, msg, message, "Skipping because a more recent info report already exists for this cluster")
+			return nil
```

Comment:
No, when we skip a message we don't treat it as an error, just as something that should be logged. The same is done for the main report (probably it is not possible to reach this part of the code, but just in case)